scio-test/src/main/scala/com/spotify/scio/testing/PipelineTestUtils.scala

```diff
@@ -124,14 +124,23 @@ trait PipelineTestUtils {
   : Seq[U] = {
     runWithLocalOutput { sc =>
       fn(sc.parallelize(data1), sc.parallelize(data2), sc.parallelize(data3), sc.parallelize(data4))
-    }
+    }._2
   }
 
-  private def runWithLocalOutput[U](fn: ScioContext => SCollection[U]): Seq[U] = {
+  /**
+   * Test pipeline components with materialized resulting collection.
```

Comment:
For consistency with the other `runWith*` methods - `Test pipeline components with a [[ScioContext]].`